pkg/ddc/alluxio/ufs_internal.go

```diff
@@ -315,6 +316,11 @@ func (e *AlluxioEngine) genUFSMountOptions(m datav1alpha1.Mount, SharedOptions m
 func (e *AlluxioEngine) genEncryptOptions(EncryptOptions []datav1alpha1.EncryptOption, mOptions map[string]string, name string) (map[string]string, error) {
 	for _, item := range EncryptOptions {
 
+		if _, ok := mOptions[item.Name]; ok {
+			err := fmt.Errorf("the name %s is duplicated with the option", item.Name)
```

Comment:
I suggest advising the end user directly "the option %s is set more than two times, please double check the dataset's option and encryptOptions."

Reply:
ok

Reply:
I think "more than one times", not "two times", right?

Reply:
I agree with "more than one times".